tokens/tokens.md

```diff
@@ -0,0 +1,99 @@
+# Native Tokens
+
+A native tokens is made up of two parts: a Native token type and a native token balance.
+
+Native tokens types are a first class object on the chain that defines the
+behaviour of a type of tokens. Native tokens live in their own account balances
+of accounts.
+
+Native tokens (from now on just tokens) are:
+- Transferable (`spend`, `trade`)
+- Countable
+- Mintable (`mint`)
```

Comment:
Who can issue mint transaction? I imagine, token issuers would like to customize minting for their needs.